docker/image_analyzer.Dockerfile

```diff
@@ -0,0 +1,102 @@
+ARG PYTHON_VERSION="3.9.13"
+ARG PLATFORM_NAME="linux/amd64"
+
+FROM --platform=${PLATFORM_NAME} python:${PYTHON_VERSION}
+
+# --- SYSTEM ARCHITECTURE
+ARG TARGETPLATFORM
+ARG TARGETARCH
+ARG TARGETVARIANT
+
+RUN printf "I'm building for TARGETPLATFORM=${TARGETPLATFORM}" \
+    && printf ", TARGETARCH=${TARGETARCH}" \
+    && printf ", TARGETVARIANT=${TARGETVARIANT} \n" \
+    && printf "With uname -s : " && uname -s \
+    && printf "and  uname -m : " && uname -mm
+
+# --- Environment variables
+ENV REQUIREMENTS_FILE="requirements.txt"
+ENV OUTDIR="/root"
+ENV PROJECT_DIR="/opt/ml"
+ENV PROGRAM_DIR="/opt/program"
+ENV HOME_DIR="/root/ml"
+ENV LOCAL_DEV_DIR="docker"
+ENV ALIASES_FILE="/root/aliases.sh"
+ENV DEBIAN_FRONTEND=noninteractive
+
+# --- Dockerfile Metadata
+LABEL Maintainer="Victor Calderon and Travis Craft"
+
+# ------------------------- COPYING AND DIRECTORIES ---------------------------
+
+RUN mkdir -p ${HOME_DIR}
+
+COPY ./src ${PROJECT_DIR}/src
+COPY ${LOCAL_DEV_DIR}/aliases.sh ${ALIASES_FILE}
+
+COPY ${REQUIREMENTS_FILE} "${HOME_DIR}/${REQUIREMENTS_FILE}"
+
+# ---------------------- EXPOSING PORTS FOR APP -------------------------------
+
+EXPOSE 80
+
+# --------------------- INSTALLING EXTRA PACKAGES -----------------------------
+# --- Updating packages and installing packages at the system-level
+
+RUN apt-get -y update && \
+    apt-get upgrade -y && \
+    apt-get clean && \
+    # Instaling system-level packages
+    apt-get install -y \
+    git \
+    ssh \
+    tree \
+    git-flow \
+    tmux \
+    direnv \
+    bash-completion \
+    zsh \
+    htop \
+    && \
+    # Cleaning out
+    rm -rf /var/lib/apt/lists/* && \
+    # Cleaning installs
+    apt-get clean && \
+    # Installing ZSH and OhZSH
+    sh -c "$(curl -fsSL https://raw.github.com/ohmyzsh/ohmyzsh/master/tools/install.sh)" && \
+    echo "source /etc/profile.d/bash_completion.sh" >> /root/.bashrc && \
+    echo "source /etc/profile.d/bash_completion.sh" >> /root/.zshrc && \
+    echo "source /root/aliases.sh" >> "${OUTDIR}/.zshrc" && \
+    echo "source /root/aliases.sh" >> "${OUTDIR}/.bashrc" && \
+    # Install direnv
+    echo 'eval "$(direnv hook zsh)"' >> "${OUTDIR}/.zshrc" && \
+    echo 'eval "$(direnv hook bash)"' >> "${OUTDIR}/.bash"
+
+# -------------------------- DOCKER-SPECIFIC ----------------------------------
+
+RUN apt-get update -y && \
+    cd ${OUTDIR_DOCKER} && \
+    curl -fsSL https://get.docker.com -o get-docker.sh && sh get-docker.sh
+
+# --------------------------- PYTHON-RELATED-LOCAL ----------------------------
+
+RUN pip3 install --upgrade pip && \
```

Comment:
Should we swap `pip3` to `pip`? I noticed the other change that did that in the Makefile.

Reply:
That's correct. Fixed.